### PR TITLE
Use short form commit hash in the gh-pages build

### PR DESCRIPTION
**Pull request by Denton-L**
As pertaining to ethereum/solidity#952.

### CODE REVIEW

**Comment by axic** [comment]
Basically this is reverting #159.

**Comment by chriseth** [comment]
What does this PR do? `git rev-parse --verify HEAD` returns the long format as does `$TRAVIS_COMMIT`.

File: ci/deploy_from_travis.sh [hunk 1]
```diff
@@ -2,18 +2,20 @@
 
 set -e
 
+SHA=`git rev-parse --verify HEAD`
```
Comment (axic) [inline ci/deploy_from_travis.sh]: For short form, shouldn't this be `git rev-parse --short --verify HEAD`?

Comment (Denton-L) [inline ci/deploy_from_travis.sh]: Thanks!

**Comment by Denton-L** [comment]
My mistake, I forgot the `--short`.

**Pull request merged**